dashboard/app/models/ability.rb

```diff
@@ -95,6 +95,7 @@ def initialize(user)
       can :create, Activity, user_id: user.id
       can :create, UserLevel, user_id: user.id
       can :update, UserLevel, user_id: user.id
+      can :create, StudentWorkEvaluation, user_id: user.id
       can :create, UserLevelEvaluation, user_id: user.id
```

Comment:
- [ ] figure this out because the current_user could also be the teacher and that would be fine too, 
- [ ] what about requestor_id? it's the student_id when the student requests evaluation of their own work and teacher_id when the teacher requests the evaluation